mease_elabftw/experiments.py

```diff
@@ -29,6 +48,7 @@ def upload_file(experiment_id, filename):
             )
             return upload_id
         else:
+            # potentially improve this error message if status.get("result") reports more information?
```

Comment:
```suggestion

```
good suggestion - but ideally this would be an issue rather than adding a comment in the code